taskcluster/tc-node_tflite-tests-prod.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -xe
+
+source $(dirname "$0")/tc-tests-utils.sh
+
+nodever=$1
+
+if [ -z "${nodever}" ]; then
+    echo "No node version given, aborting."
+    exit 1
+fi;
+
+model_source=${DEEPSPEECH_PROD_MODEL//.pb/.tflite}
```

Comment:
This needs a similar override for the URL as cpp-tflite-tests right?

Reply:
It would, but right now we don't run NodeJS prod tests at all. Should we ? I think we should.